src/foam/nanos/notification/SendEmailSetting.js

```diff
@@ -0,0 +1,53 @@
+foam.CLASS({
+  package: 'foam.nanos.notification',
+  name: 'SendEmailSetting',
```

Comment:
Why do we call these send...  why not just EmailSettings and NotificationSettings?
And the NotificationSettings has send(x, user, notification).